DevCycle/DVCClient.swift

```diff
@@ -128,6 +131,19 @@ public class DVCClient {
         self.options = options
     }
     
+    func refetchConfig() {
+        if (self.initialized) {
+            self.service?.getConfig(user: self.lastIdentifiedUser!, enableEdgeDB: self.enableEdgeDB, completion: { [weak self] config, error in
```

Comment:
usually it's safer to unwrap optionals like this instead of using the `!` operator:

```
if let lastIdentifiedUser = self.lastIdentifiedUser, self.initialized {
    self.service?.getConfig(user: lastIdentifiedUser, ...)
}
```

Reply:
is it even possible for lastIdentifiedUser to not be set? Shouldn't it just not be optional?

Reply:
it has to be optional since the builder creates an empty `DVCClient`. Otherwise you have to set it in its `init` function (not to be confused with the `initialize` function we defined that gets called when a user calls `.build()`.